### DISCUSSION TITLE
How to get internationalized content for the nav links?

### DISCUSSION THREAD

**Discussion by jpmti2016** [fuma-nama/fumadocs#1209] [Q&A]
My `baseOptions` have a structure like this: 

 nav: {
    title: "Back to Fluency",
  },
  links: [
    {
      text: "Courses",
      url: "/courses",
      active: "nested-url",
    },
]

For 'es' I would like to have: 
  nav: {
    title: "Fluente de nuevo",
  },
  links: [
    {
      text: "Cursos",
      url: "es/courses",
      active: "nested-url",
    },

And so on for other languages. 

Can I do this with the `baseOptions` config or have to pass a custom `nav` to `HomeLayout` and `DocsLayout`?

**Comment by fuma-nama** [comment]
You can change the passed config based on `params.lang`, they are just props
